event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatcherProperties.java

```diff
@@ -25,6 +25,7 @@
 public class EventDispatcherProperties {
 
 	private String namespace = "default";
+	private String sidecarImageName = "sk8s/function-sidecar";
```

Comment:
we could leave this out if providing via `application.properties` (or if we're going to have a default here, then we don't need `application.properties` anyways, but this should also include the tag)